sqlite_fdw.c

```diff
@@ -3932,16 +3934,18 @@ sqlite_foreign_grouping_ok(PlannerInfo *root, RelOptInfo *grouped_rel)
 	 */
 	if (fpinfo->local_conds)
 	{
+		List	   *aggvars1 = NIL;
```

Comment:
Could you make same as postgres_fdw:  
- remove definition in L3759
- add new definition in L3854
- revert the changed here.

Reply:
In my current git file state point there is other line numbers, @nxhai98 . I have changed some near lines in https://github.com/pgspider/sqlite_fdw/pull/88/commits/08d87482359053057ff47d4418fbd265069a5a25 and have no failed multiversional tests. Please verify current state and let tell me where where is incorrect declarations now.

Reply:
Thank you. It's same as my expectation.
Please help me to resolve this thread.